lib/std/fmt.zig

```diff
@@ -467,6 +467,28 @@ pub fn formatType(
             });
             return formatType(@as(Slice, &value), fmt, options, context, Errors, output, max_depth);
         },
+        .Vector => {
+            const len = @typeInfo(T).Vector.len; // TODO .len for vectors
```

Comment:
This is not a TODO; `len` for vectors is not planned. Vectors can be vectors of single-item pointers to structs, and `.len` will do a vectorized field access. #363

Reply:
https://github.com/ziglang/zig/issues/903#issuecomment-459508820

> - [ ] len property on the type

?

Reply:
This would be e.g. `@Vector(4, i32).len`, much like `u32.bit_count`. However I'm not sure why I added that because both of those things are deprecated in favor of `@typeInfo`. I will remove the checkbox.